### PR TITLE
gpio-ich/itco-wdt/w83627hf-wdt: add modules for Xeon platforms

### PR DESCRIPTION
**Pull request by pprindeville**
Drivers for supporting current Xeon motherboards.

Signed-off-by: Philip Prindeville <philipp@redfish-solutions.com>

### CODE REVIEW

**Comment by pprindeville** [comment]
@diizzy, @nbd168, @pkgadd

File: package/kernel/linux/modules/other.mk [hunk 1]
```diff
@@ -1129,3 +1129,52 @@ endef
...
+define KernelPackage/gpio-ich
+  TITLE:= Intel ICH GPIO support
+  SUBMENU:=$(OTHER_MENU)
+  KCONFIG:= \
+    CONFIG_GPIO_ICH
+  FILES:= \
+    $(LINUX_DIR)/drivers/gpio/gpio-ich.ko
+  AUTOLOAD:= $(call AutoProbe, gpio-ich)
+  DEPENDS:=@GPIO_SUPPORT
+endef
```
Comment (seragh) [inline package/kernel/linux/modules/other.mk]: Please use the same format as for itco-wdt below wrt to ordering, spacing and line breaks.

**Comment by pprindeville** [comment]
Hold off on this... for whatever reason, the /etc/modprobe.d/ file isn't being generated.  Investigating...

File: package/kernel/linux/modules/other.mk (hunk 1 above)
Comment (pprindeville) [inline package/kernel/linux/modules/other.mk]: Fixing and building.. will re-push shortly.

**Comment by pprindeville** [comment]
Okay, fixed the issue.  The package name has to exactly match the driver basename, no matter how ugly...

**Comment by pprindeville** [comment]
Tested by building an image, burning it onto a CFast, and rebooting.

All modules load correctly.

File: package/kernel/linux/modules/other.mk (hunk 1 above)
Comment (seragh) [inline package/kernel/linux/modules/other.mk]: I meant the other way around fix gpio-ich to be like itco-wdt, so it looks like the surrounding code.

**Comment by wigyori** [comment]
Intel TCO Timer/Watchdog Specific Vendor Support (ITCO_VENDOR_SUPPORT) [N/y/?] (NEW) 

This one seems to be missing from the iTCO_wdt package.

**Comment by pprindeville** [comment]
> This one seems to be missing from the iTCO_wdt package.

Fixed.

**Comment by wigyori** [comment]
Great, thanks!

**Comment by pprindeville** [comment]
Thanks!

**Pull request merged**